src/Propel/Generator/Model/Column.php

```diff
@@ -958,7 +958,7 @@ public function isPrimaryString(): bool
      *
      * @return void
      */
-    public function setPrimaryKey($flag = true): void
+    public function setPrimaryKey(bool $flag = true): void
```

Comment:
No defaults on setters.

Reply:
This caused quite a few additional tests to fail
Given the BC impact I am a little less sure about this one now.

But I can see why having it not defaulting seems OK moving forward since the naming does not clearly indicate a positive default value (useAsPrimaryKey etc would be a different story).